source/MaterialXShaderGen/ShaderGenerators/Glsl/GlslSyntax.cpp

```diff
@@ -259,6 +259,18 @@ GlslSyntax::GlslSyntax()
         )
     );
 
+    addTypeSyntax
+    (
+        DataType::LIGHT,
```

Comment:
The data type output from light shaders. Holds intensity and direction results from sampling the light source.